public/stylesheets/navbar.css

```diff
@@ -0,0 +1,108 @@
+/* General Navbar Styles */
+.navbar {
+    background-color: black; /* Black background for the navbar */
+    transition: background-color 0.3s ease-in-out;
```

Comment:
what is the use of this transition?

Reply:
Smooth transition for background color when it changes, enhancing user experience

Reply:
When exactly is the background color supposed to change? It currently stays black all the time.

Reply:
Actually i wrote this when initially i changed the color to white to get a transition. Now its not required so i removed it. Thanks for the feedback !